### ISSUE TITLE
TiUP无法编辑prometheus相关配置文件，如各告警规则配置文件

### ISSUE THREAD

**Issue by topbrightwen** [pingcap/tiup#604]
TiDB升级到4.0后，改用TiUP来管理集群，但发现TiUP无法编辑prometheus相关配置文件，如各种告警规则配置文件，给管理造成很大不便，希望能增加这项功能。

**Comment by lucklove** [comment]
I think we can add an env variable to avoid covering user-edited configs in the [startup script](https://github.com/pingcap/tiup/blob/master/templates/scripts/run_prometheus.sh.tpl#L10):
```
if [ $DONT_COVER_CONFIG == "true" ]; do
    cp {{.DeployDir}}/bin/prometheus/*.rules.yml {{.DeployDir}}/conf/
done
```

The variable can be added into the systems config file as [this style](https://coreos.com/os/docs/latest/using-environment-variables-in-systemd-units.html):

```
monitoring_servers:
  - host: x.x.x.x
    env:
        - DONT_COVER_CONFIG: true
```

@lonng What do you think about this?

**Comment by lucklove** [comment]
Fixed by https://github.com/pingcap/tiup/pull/712

**Comment by lonng** [comment]
> Fixed by #712

@lucklove Better to link the documentation.
